src/Entity/RealmSsid.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of letswifi; a system for easy eduroam device enrollment
+ * Copyright: 2023, Paul Dekkers, SURF <paul.dekkers@surf.nl>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+namespace App\Entity;
+
+use App\Repository\RealmSsidRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: RealmSsidRepository::class)]
+class RealmSsid
+{
+    #[ORM\Id]
+    #[ORM\OneToOne(inversedBy: 'realmSsid', cascade: ['persist', 'remove'])]
```

Comment:
Ben ergens wel nieuwsgierig waarom ervoor gekozen is om one-to-one's te gebruiken, kom die in de praktijk niet zo vaak voor omdat de attribs vaak gewoon "in" de table genormaliseerd worden.